security/storage.adoc

```diff
@@ -78,7 +79,7 @@ visible to the container with it is associated.
 
```

Comment:
The above says "...with it is associated." Should that be "...with which it is associated?"